extensions/amp-form/0.1/amp-form.js

```diff
@@ -258,6 +262,29 @@ export class AmpForm {
     return url;
   }
 
+  /**
+   * Gets and validates an attribute for form request encoding type.
+   * @param {string} attribute
+   * @return {?string}
+   * @private
+   */
+  getEncType_(attribute) {
+    const encType = this.form_.getAttribute(attribute);
+    if (
+      encType === 'application/x-www-form-urlencoded' ||
+      encType === 'multipart/form-data'
+    ) {
+      return encType;
+    }
+    if (encType !== null) {
```

Comment:
nit: could this be an `else if` instead?

Reply:
Yep, I have updated it to `else if`